cmd/kubeadm/app/preflight/checks.go

```diff
@@ -841,6 +841,61 @@ func (ResolveCheck) Check() (warnings, errors []error) {
 	return warnings, errors
 }
 
+const (
+	// IPVSProxyMode is match set up cluster with ipvs proxy model
+	IPVSProxyMode = "ipvs"
+)
+
+// Sets of IPVS required kernel modules.
+var ipvsModules = []string{
+	"ip_vs",
+	"ip_vs_rr",
+	"ip_vs_wrr",
+	"ip_vs_sh",
+	"nf_conntrack_ipv4",
+}
+
+// KernelModuleIPVSRequiredCheck tests IPVS required kernel moduler.
+type KernelModuleIPVSRequiredCheck struct{}
+
+// Name returns label for KernelModuleIPVSRequiredCheck
+func (KernelModuleIPVSRequiredCheck) Name() string {
+	return "KernelModuleIPVSRequired"
+}
+
+// LinuxKernelHandler implements KernelHandler interface.
+type LinuxKernelHandler struct {
+	executor utilsexec.Interface
+}
+
+// Check try to validates IPVS required kernel modules exists or not.
+func (KernelModuleIPVSRequiredCheck) Check() (warnings, errors []error) {
+	glog.V(1).Infoln("validating the kernel module IPVS required exists in Master/Node or not")
+
+	warnings = []error{}
+	// Find out loaded kernel modules
+	handle := &LinuxKernelHandler{}
+	out, err := handle.executor.Command("cut", "-f1", "-d", " ", "/proc/modules").CombinedOutput()
+	if err != nil {
+		errors = append(errors, fmt.Errorf("error getting installed ipvs required kernel modules: %v", err))
```

Comment:
This matches the existing kube-proxy check for ipvs, but will not detect ipvs support that is compiled into the kernel rather than built as a module: https://github.com/kubernetes/kubernetes/issues/63801

Reply:
Fixed